app/views/common/_obsolete_browsers.html.erb

```diff
@@ -0,0 +1,13 @@
+<div id="outdated">
```

Comment:
I wonder if this should be in rails at all? after all the layout has already moved to react... we can simply add it to our layout view? /cc @glekner

Reply:
why not use `mount_react_component` ?